### DISCUSSION TITLE
Immudb support

### DISCUSSION THREAD

**Discussion by jayket** [Budibase/budibase#10426] [Help]
Has anyone gotten immudb to work as a data source for budibase? When I try to use it I get an error something along the lines of unexpected dot at position 33. I’m trying to make a web app to make a notary ejournal any suggestions would be really helpful. Thanks for the help.

**Comment by andz-bb** [comment]
Hey @jayket 

Could you give some more info on how you are currently attempting this with Budibase? Are you using [immugw](https://github.com/codenotary/immugw) and then connecting to that with a REST datasource? I am not too familiar with immudb but I will try to help you get going with it in Budibase. 

If you have trouble getting it working with the REST API connector it could be a good candidate for a [custom datasource plugin.](https://docs.budibase.com/docs/custom-datasource)

**Comment by jayket** [comment]
I was using postgresql with port 5432. 
I was trying to set up with rest but honestly have no idea about that sorry

**Reply by andz-bb** [reply]
@jayket I tested it a bit yesterday to see if I could get it to connect with the postgres data source, but unfortunately I wasn't able to get it to work. I think a custom data source plugin could be required if you're unable to get the REST API working. I will have a closer look next week if I get the chance!
